src/plot_api/plot_api.js

```diff
@@ -1374,7 +1374,8 @@ function _restyle(gd, aobj, _traces) {
         'line.cmin', 'line.cmax',
         'marker.line.cmin', 'marker.line.cmax',
         'contours.start', 'contours.end', 'contours.size',
-        'contours.showlines',
+        'contours.showlines', 'contours.showlabels', 'contours.labelformat',
```

Comment:
Wanna try adding `editType` flags their attribute declarations instead as in https://github.com/plotly/plotly.js/pull/1653 ?